docs/modules/ROOT/pages/get-started.adoc

```diff
@@ -101,75 +77,69 @@ Members {size:3, ver:3} [
 ]
 ```
 
-== Connect to Hazelcast from Outside Kubernetes
+== Start Hazelcast Enterprise Cluster
 
-To connect to a Hazelcast cluster from outside Kubernetes, you need to configure the `expose-externally` field of your custom resource.
+Hazelcast Enterprise requires having the license key specified as Kubernetes Secret.
 
+NOTE: If you don't have Hazelcast Enterprise License Key, please check xref:deploy:using-enterprise-edition.adoc#requesting-a-license-key[Requesting a License Key]
```

Comment:
```suggestion
NOTE: If you don't have Hazelcast Enterprise License Key, please check xref:hazelcast:deploy:using-enterprise-edition.adoc#requesting-a-license-key[Requesting a License Key]
```

Reply:
Now that these docs are in their own subsite (Antora component), we need to reference the `hazelcast` component to link to the Platform docs.

Reply:
I updated the link to go straight to the Hazelcast website. The docs are just an extra step that users don't need.